ACE/ace/INET_Addr.cpp

```diff
@@ -290,15 +291,17 @@ ACE_INET_Addr::set (const char address[], int address_family)
 }
 
 ACE_INET_Addr::ACE_INET_Addr (const char address[], int address_family)
-  : ACE_Addr (determine_type (), sizeof (inet_addr_))
+  : ACE_Addr (determine_type (), sizeof (inet_addr_)),
+    if_name_( 0 )
 {
   ACE_TRACE ("ACE_INET_Addr::ACE_INET_Addr");
   this->set (address, address_family);
 }
 
 #if defined (ACE_HAS_WCHAR)
 ACE_INET_Addr::ACE_INET_Addr (const wchar_t address[], int address_family)
-  : ACE_Addr (determine_type (), sizeof (inet_addr_))
+  : ACE_Addr (determine_type (), sizeof (inet_addr_)),
+    if_name_( 0 )
```

Comment:
nullptr

Reply:
Just submitted a refactor to not use `ACE_OS` fns and `char *` for this member variable.